codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/containsreservedkeyword.java

```diff
@@ -0,0 +1,193 @@
+
+    public static Class<? extends Builder> serializableBuilderClass() {
+        return BuilderImpl.class;
+    }
+
+    @Override
+    public final int hashCode() {
+        int hashCode = 1;
+        hashCode = 31 * hashCode + Objects.hashCode(nul());
+        return hashCode;
+    }
+
+    @Override
+    public final boolean equals(Object obj) {
+        return equalsBySdkFields(obj);
+    }
+
+    @Override
+    public final boolean equalsBySdkFields(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (!(obj instanceof ContainsReservedKeyword)) {
+            return false;
+        }
+        ContainsReservedKeyword other = (ContainsReservedKeyword) obj;
+        return Objects.equals(nul(), other.nul());
+    }
+
+    /**
+     * Returns a string representation of this object. This is useful for testing and debugging. Sensitive data will be
+     * redacted from this string using a placeholder value.
+     */
+    @Override
+    public final String toString() {
+        return ToString.builder("ContainsReservedKeyword").add("NUL", nul()).build();
+    }
+
+    public final <T> Optional<T> getValueForField(String fieldName, Class<T> clazz) {
+        switch (fieldName) {
+            case "NUL":
+                return Optional.ofNullable(clazz.cast(nul()));
+            default:
+                return Optional.empty();
+        }
+    }
+
+    @Override
+    public final List<SdkField<?>> sdkFields() {
+        return SDK_FIELDS;
+    }
+
+    @Override
+    public final Map<String, SdkField<?>> sdkFieldNameToField() {
+        return SDK_NAME_TO_FIELD;
+    }
+
+    private static Map<String, SdkField<?>> memberNameToFieldInitializer() {
+        Map<String, SdkField<?>> map = new HashMap<>();
+        map.put("NULL", NUL_FIELD);
+        return Collections.unmodifiableMap(map);
+    }
+
+    private static <T> Function<Object, T> getter(Function<ContainsReservedKeyword, T> g) {
+        return obj -> g.apply((ContainsReservedKeyword) obj);
+    }
+
+    private static <T> BiConsumer<Object, T> setter(BiConsumer<Builder, T> s) {
+        return (obj, val) -> s.accept((Builder) obj, val);
+    }
+
+    @Mutable
+    @NotThreadSafe
+    public interface Builder extends SdkPojo, CopyableBuilder<Builder, ContainsReservedKeyword> {
+        /**
+         * Sets the value of the NUL property for this object.
+         *
+         * @param nul
+         *        The new value for the NUL property for this object.
+         * @return Returns a reference to this object so that method calls can be chained together.
+         */
+        Builder nul(String nul);
+    }
+
+    static final class BuilderImpl implements Builder {
+        private String nul;
+
+        private BuilderImpl() {
+        }
+
+        private BuilderImpl(ContainsReservedKeyword model) {
+            nul(model.nul);
+        }
+
+        public final String getNul() {
```

Comment:
Do we need getNull?

Reply:
No, not for this particular issue. Actually, adding `getNull()` could be problematic, because if you went from `AttributeValue.Builder` -> JSON, you'll get both `nul` and `null` in the output JSON, which could be problematic

```json
{
  "nul": true,
  "null": true
}
```